src/shared/particle_dynamics/fluid_dynamics/fluid_integration.hpp

```diff
@@ -71,7 +71,7 @@ void Integration1stHalf<Inner<>, RiemannSolverType, KernelCorrectionType>::inter
         Real dW_ijV_j = inner_neighborhood.dW_ijV_j_[n];
         const Vecd &e_ij = inner_neighborhood.e_ij_[n];
 
-        force -= mass_[index_i] * (p_[index_i] * correction_(index_i) + p_[index_j] * correction_(index_j)) * dW_ijV_j * e_ij;
+        force -= mass_[index_i] * (p_[index_i] * correction_(index_j) + p_[index_j] * correction_(index_i)) * dW_ijV_j * e_ij;
```

Comment:
@WeiyiVirtonomy Should we backport those changes ?

Reply:
Sorry. I will chnage it back. This form will be tested latter in another branch.